python/llm/example/CPU/HF-Transformers-AutoModels/Model/falcon/generate.py

```diff
@@ -43,11 +43,11 @@
     # which convert the relevant layers in the model into INT4 format
     model = AutoModelForCausalLM.from_pretrained(model_path,
                                                  load_in_4bit=True,
-                                                 trust_remote_code=True)
+                                                 trust_remote_code=False)
```

Comment:
Can setting "trust_remote_code=False" work for transformer==4.31.0? Otherwise we can keep "trust_remote_code=True" and just add note for transformer==4.36.0

Reply:
"trust_remote_code=False" works for transformer==4.31.0